src/NextjsBuild.ts

```diff
@@ -123,21 +93,18 @@ export class NextjsBuild extends Construct {
     }
 
     // build environment vars
-    const outputTracingRoot = this.projectRoot;
     const buildEnv = {
       ...process.env,
-      [NEXTJS_BUILD_STANDALONE_ENV]: 'true',
-      [NEXTJS_BUILD_OUTPUTTRACEROOT_ENV]: outputTracingRoot,
       ...getBuildCmdEnvironment(this.props.environment),
       ...(this.props.nodeEnv ? { NODE_ENV: this.props.nodeEnv } : {}),
     };
 
     const buildPath = this.props.buildPath ?? nextjsPath;
-    const packageManager = this.props.packageManager ?? 'npm';
-    const buildCommand = packageManager === 'yarn' ? ['build'] : ['run', 'build'];
+    const buildCommand = this.props.buildCommand ?? 'npx --yes open-next@latest build';
```

Comment:
Maybe let people pin this version in pkg json?

Reply:
Do you mean `npx --yes open-next build`? I don't know how much `npx` respects package.json